README.md

```diff
@@ -22,79 +22,199 @@ This version supports serverless functions on AWS (Lambda functions) only, suppo
-    select post verification action : (1) for detect; (2) for block; leave empty for no post verification action to perform
-    enter SNS arn if you would like to be notified when signature verification fails, otherwise press enter:
-    is there existing trail in CloudTrail (in the region selected above) which you would like to use? (if no, please press enter):
-    do you want to work in keyless mode (y/n): n
-    enter path to custom public key for code signing? (if you want us to generate key pair, please press enter):
-    Enter password for private key:
-    Enter password for private key again:
-    Private key written to cosign.key
-    Public key written to cosign.pub
-    Uploading function-clarity function code to s3 bucket, this may take a few minutes
-    function-clarity function code upload successfully
-    deployment request sent to provider
-    waiting for deployment to complete
-    deployment finished successfully
-```
-
-### Sign function code
-Use the command below to sign a folder containing function code, and then upload it to the user cloud account.
+- Download the latest `aws_function.tar.gz` and `functionclarity-vYYY<os_type>` files from: https://github.com/openclarity/functionclarity/releases
 
-```shell
-./functionclarity sign aws code /sample-code-verified-folder
+- Extract both files into the same directory:
+  ```
+  # tar -xzvf aws_function.tar.gz
+  x aws_function
 
-using config file: /Users/john/.fc
+  # tar -xzvf functionclarity-v1.0.2-darwin-amd64.tar.gz
+  x functionclarity unified
+  x unified-template.template
+  ```
+  Other than the original gz files, there should be the following files in the same directory:
+  ```
+  aws_function
+  functionclarity
+  unified-template.template
+  ```
+**Note: Depending on the operating system that you are using to run the `functionclarity` binary, you may have to trust the binary. In MacOS, a warning will popup on first use of the functionclarity command. It will look similar to this:
+
+![image](https://user-images.githubusercontent.com/3701244/207752284-f525db61-f1ff-46d5-9a40-d8a85982f0ef.png)
+
+On a Mac, you can resolve this issue by going to Settings > Security & Privacy > Privacy > Developer Tools and checking the box next to "functionclarity" as shown here:
+
+![image](https://user-images.githubusercontent.com/3701244/207755079-688c7a83-6e7b-4c5c-ba06-33c8fbe4aea8.png)
+
+Go into Finder and locate the `functionclarity` binary and right-click on the file and click "open". Another popup will appear that looks like this:
+
+![image](https://user-images.githubusercontent.com/3701244/207755405-9d784f33-f046-4955-bacc-fb4acc54f251.png)
+
+A new terminal window will open and the command will run one time. You can close this terminal window if you use another terminal type like iTerm2 or you can leave it open for the additional FunctionClarity command-line steps below.
+
+## AWS IAM Role / Policy Creation
```

Comment:
@pkalapat what you intended to do overhere?

Reply:
The intention is that when you do the very first approval of the functionclarity binary run, it opens in the default terminal on your machine. That terminal may not be the one you typically use. It is, again, a way to ease the friction for interacting with a new project and an untrusted/unsigned binary (like the Mac approval steps above this section). If you believe that people don't need help with the consumption of your unsigned binary, then we can consider removing this information or stuffing it in a 'reference' section.

Reply:
Rule #1 of easing consumption of a new project, never assume anyone knows anything about using your project. :-)

Reply:
I have to say... I don't agree with you about that....
If you are going to try an opensource solution - it means that you have an idea about how to run things on your comupter and what your capabilities are. when you get into an opensource that handles servereless functions - we assume that you are familliare with serverless frameworks and you handle serverless functions on you their day to day.

Yes - there is improvements that needs to be added to the ReadMe.  Yes we need to place somewhere what is the minimum permissions that you should have in order to run the solution. But I don't think that a specific instructions for Mac is something that belongs to the ReadMe. maybe FAQs? 


For example take a look at github cli (and you can find many more) and their instruction https://github.com/cli/cli/blob/trunk/docs/install_linux.md. The first paragraph leads you to their release page ( like we have ) and you can try to download their binary. Yes you will need to go through the same procedure that @pkalapat mentioned - but they are not specifing that.